api/javatests/org/openyolo/api/CredentialClientTest.java

```diff
@@ -295,31 +293,34 @@ public void getSaveIntent_noKnownProviders() throws Exception {
     }
 
     @Test
-    public void getCredentialFromActivityResult_noExtras() throws Exception {
+    public void testGetCredentialRetrieveResult() throws Exception {
```

Comment:
Old habits die hard; the test prefix was JUnit 3 and even though we now use the Test annotation, I still reflexively write the test prefix. I can remove in a follow-up that removes this from all tests.

I like your template, we can move to that.